src/indexing/Elsa.Indexing.Abstractions/Extensions/ElsaIndexingOptionsExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Reflection;
+
+using MediatR;
+
+namespace Elsa.Indexing.Extensions
+{
+    public static class ElsaIndexingOptionsExtensions
+    {
+        public static ElsaOptions UseIndexing(this ElsaOptions options, Action<ElsaIndexingOptions> configure)
+        {
+            var indexingOptions = new ElsaIndexingOptions(options.Services);
+            configure.Invoke(indexingOptions);
+
+            options.Services.AddMediatR(Assembly.GetExecutingAssembly());
```

Comment:
Please remind me; was it safe to invoke `AddMediatR` like this multiple times?

Reply:
You are right. Is corrected in the next commit.